subset/brute/report.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
what's this file for?

Reply:
removed file, leftover from development testing